content/day.css

```diff
@@ -0,0 +1,91 @@
+html, body {
+    height: 100%;
+    width: 100%;
+    margin: 0;
+    overflow:hidden;
+	background-color:#e0e0e0;
+	color:#000000;
+    font-family: Garamond, Palatino, Palladio, serif;
```

Comment:
Why do you put all of this? We are only interested in `color` and `background-color`, right?